lang/c++/test/CodecTests.cc

```diff
@@ -25,7 +25,7 @@
 #include "Specific.hh"
 #include "ValidSchema.hh"
 
-#include <boost/bind.hpp>
+#include <boost/bind/bind.hpp>
```

Comment:
Recent versions of boost have effectively deprecated this header, we don't use placeholders, so no additional changes are required:

```
In file included from /usr/include/boost/bind.hpp:30,
                 from /home/gerrit/Desktop/avro/lang/c++/test/CodecTests.cc:28:
/usr/include/boost/bind.hpp:36:1: note: ‘#pragma message: The practice of declaring the Bind placeholders (_1, _2, ...) in the global namespace is deprecated. Please use <boost/bind/bind.hpp> + using namespace boost::placeholders, or define BOOST_BIND_GLOBAL_PLACEHOLDERS to retain the current behavior.’
   36 | BOOST_PRAGMA_MESSAGE(
      | ^~~~~~~~~~~~~~~~~~~~
```